docs/reference/symbols.rst

```diff
@@ -47,6 +47,7 @@ are inherited by the :ref:`model symbols <symbols_model_symbols>`.
 
     ~ArraySymbol.all
     ~ArraySymbol.any
+    ~ArraySymbol.concatenate
```

Comment:
I think we want to add it as a function in the `mathematical.py` rather than as a method. Function should have a similar signature to [np.concatenate()](https://numpy.org/doc/stable/reference/generated/numpy.concatenate.html).

Reply:
fixed!